### ISSUE TITLE
Add a whitelist for repo launch limits on try.jupyter.org

### ISSUE THREAD

**Issue by choldgraf** [jupyterhub/mybinder.org-deploy#803]
I noticed recently that the "introduction to IPython" repository was at the launch limit on Binder. Since these are repositories that we host, and that we want to be maximally available (since they'll be hit by new users most likely), what do folks think about lifting the launch limit on these repositories only?

**Comment by minrk** [comment]
Makes sense, but I also like having an upper limit on the cost that can come from these repos. These demo repos are the ones I feel the *least* bad about saying "Sorry, we're busy", as long as the experience of being full is clear and friendly enough.
